### ISSUE TITLE
why we set eos_token and bos_token to be the same?

### ISSUE THREAD

**Issue by vanpersie32** [salesforce/LAVIS#335]
https://github.com/salesforce/LAVIS/blob/59273f651b9bffb193d1b12a235e909e9f826dda/lavis/models/blip2_models/blip2_vicuna_instruct.py#L86C1-L89

**Comment by ustcwhy** [comment]
Do you understand why bos token is set as </s> ? Do you try to use Bos token as <s> for InstructBLIP with vicuna-7b?

**Comment by vanpersie32** [comment]
Yes, I have try to set bos token as <s>, the result is better than  that of </s>
